### ISSUE TITLE
Add a 'favicon' to the app

### ISSUE THREAD

**Issue by paulkore** [paulkore/moneytracker#20]
Currently, the word "trakr" is positioned on the left side of the top bar to serve as a logo.

This is fine, however there is no 'favicon' at all which makes it harder to see the trakr tab when multiple tabs are open in the browser.

In scope for this task is finding a suitable (free) favicon image online, as well as integrating it into the app - This has an element of research, since it might require a Django plugin.

Designing a proper favicon is out of scope, and will become relevant once the official branding is in place (the current branding is temporary).

**Comment by paulkore** [comment]
It's alive.
